server/src/controllers/__tests__/user.spec.ts

```diff
@@ -63,18 +63,18 @@ describe("UserController", () => {
   })
 
   it("should create a user", async () => {
-    const mockUser: [{ id: number; email: string }, boolean] = [{ id: 1, email: "test@test.com" }, true]
-    const mockScore: [{ userId: number; score: number }, boolean] = [{ userId: 1, score: 0 }, true]
-    const expectedFullUser = { user: mockUser[0], scores: mockScore[0] }
+    const mockUser: [{ id: number; email: string; streak: 0 }, boolean] = [
+      { id: 1, email: "test@test.com", streak: 0 },
+      true,
+    ]
+    const expectedUser = mockUser[0]
```

Comment:
Creating a user doesn't need to update scores at all anymore, only when a user is logged in and scores.

Reply:
We create a score for each day the user is logged in and plays rather than update one, very nice 🧼